### ISSUE TITLE
[11.5.0] Cannot find module '../lib/cli.js'

### ISSUE THREAD

**Issue by baixiaoyu2997** [chromaui/chromatic-cli#997]
This error only occurred in 11.5.0, and testing with 11.4.0 did not result in this error
```
The CLI tried to run your build script, but the command failed. This indicates a problem with your Storybook. Here's what to do:
- Check the Storybook build log printed below.
- Run npm run build or yarn build yourself and make sure it outputs a valid Storybook by opening the generated index.html in your browser.
- Review the build-storybook CLI options at https://storybook.js.org/docs/configurations/cli-options/#for-build-storybook
Command failed with exit code 1: pnpm run build --output-dir /tmp/chromatic--533231-bnijedBhE23m
node:internal/modules/cjs/loader:1051
  throw err;
  ^
Error: Cannot find module '../lib/cli.js'
Require stack:
- /home/ubuntu/actions-runner/externals.2.316.1/node20/bin/npm
    at Module._resolveFilename (node:internal/modules/cjs/loader:1048:15)
    at Module._load (node:internal/modules/cjs/loader:901:27)
    at Module.require (node:internal/modules/cjs/loader:1115:19)
    at require (node:internal/modules/helpers:130:18)
    at Object.<anonymous> (/home/ubuntu/actions-runner/externals.2.316.1/node20/bin/npm:2:1)
    at Module._compile (node:internal/modules/cjs/loader:1241:14)
    at Module._extensions..js (node:internal/modules/cjs/loader:1295:10)
    at Module.load (node:internal/modules/cjs/loader:1091:32)
    at Module._load (node:internal/modules/cjs/loader:938:12)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:83:12) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [ '/home/ubuntu/actions-runner/externals.2.316.1/node20/bin/npm' ]
}
Node.js v20.8.1
ℹ Build command:
pnpm run build --output-dir /tmp/chromatic--533231-bnijedBhE23m
ℹ Runtime metadata:
{
  "nodePlatform": "linux",
  "nodeVersion": "20.8.1",
  "packageManager": "pnpm",
  "packageManagerVersion": "9.1.4"
}
ℹ Storybook build output:
/home/ubuntu/actions-runner/_work/deorderbook-monorepo/deorderbook-monorepo/apps/storybook/build-storybook.log
> @deorderbook/storybook@0.0.1 build /home/ubuntu/actions-runner/_work/deorderbook-monorepo/deorderbook-monorepo/apps/storybook
> npm run prepare:base && npm run prepare:nuxt && cross-env MODE=storybook storybook build --stats-json && tsx ./scripts/copy-dist "--output-dir" "/tmp/chromatic--533231-bnijedBhE23m"
 ELIFECYCLE  Command failed with exit code 1.
    → Command failed: pnpm run build --output-dir /tmp/chromatic--533231-bnijedBhE23m
Error: non-zero exit code
```

![image](https://github.com/chromaui/chromatic-cli/assets/9149999/8951996d-57df-4518-9efd-899325c15e94)

this is my CI file:
```
# .github/workflows/chromatic.yml
name: Chromatic

on: workflow_call

concurrency:
  group: ${{ github.workflow }}-${{ github.ref }}-chromatic
  cancel-in-progress: true

permissions:
  actions: write
  contents: read

jobs:
  chromatic:
    name: Chromatic
    runs-on: self-hosted
    steps:
      - name: ⬇️ Checkout repo
        uses: actions/checkout@v4
        with:
          fetch-depth: 0
          token: ${{ secrets.PAT }}
      - uses: pnpm/action-setup@v3
        with:
          version: latest
      - name: ⎔ Setup node
        uses: actions/setup-node@v4
        with:
          cache: 'pnpm'
          node-version: 20

      - name: 📥 Install deps
        run: pnpm install

      - name: ⚡ Run chromatic
        # NOTE: do not use 11.5.0, when build has error
        uses: chromaui/action@v11.4.0
        # Chromatic GitHub Action options
        with:
          workingDir: apps/storybook
          # 👇 Chromatic projectToken, refer to the manage page to obtain it.
          projectToken: ${{ secrets.CHROMATIC_PROJECT_TOKEN }}
          exitZeroOnChanges: true
          # exitOnceUploaded: true
          # onlyChanged: true # NOTE: do not use onlyChanged with pull_request event, https://www.chromatic.com/docs/setup-turbosnap/#github-pull_request-triggers
          buildScriptName: 'build'
          autoAcceptChanges: 'develop'
          skip: '@(renovate/**|dependabot/**)'

```

**Comment by cgbl-90** [comment]
@baixiaoyu2997, thanks for reporting this issue!
Our engineers have identified the issue and released a fix already. Can you please try upgrading to `chromaui/action-canary@v11.5.0` and run a build? Use the `--debug` flag to get extra logs in case it fails.

**Comment by cgbl-90** [comment]
@baixiaoyu2997 If this workaround doesn't help, you can continue working by going back to `chromaui/action@v11.3.0` while we work on this. Waiting for your feedback

**Comment by baixiaoyu2997** [comment]
Thank you for your reply. I am in a collaborative project with multiple people, so I am unable to frequently modify it. Yesterday, I fixed it to version 11.4.0 and it is working

**Comment by cgbl-90** [comment]
@baixiaoyu2997 We have released a new CLI version. Can you try updating to [v11.5.1](https://github.com/chromaui/chromatic-cli/releases) (`chromaui/action@v11.5.1`)?
